packages/components/src/primitives/button/index.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
There's a [new package](https://github.com/WordPress/gutenberg/tree/master/packages/primitives#primitives) we can put this into, looks like a better place for this.